.github/workflows/lint.yml

```diff
@@ -4,7 +4,7 @@ on:
 permissions:
   contents: read
 jobs:
-  lint:
+  golangci:
```

Comment:
Keep `lint` is fine, otherwise, we need to change the rule names of other repos.